src/main/java/com/minecolonies/coremod/tileentities/TileEntityBarrel.java

```diff
@@ -106,15 +113,44 @@ private void doBarrelCompostTick(final Level worldIn, final BlockPos pos, final
     /**
      * Method called when a player uses the block. Takes the needed items from the player.
      *
+     * @deprecated Use side-sensitive version
      * @param playerIn  the player
      * @param itemstack the itemStack on the hand of the player
      * @return if the barrel took any item
      */
+    @Deprecated
     public boolean useBarrel(final Player playerIn, final ItemStack itemstack)
```

Comment:
While deprecating the old method when changing parameters like this is in general a good practice, there isn't anywhere else that calls the old method, so it's not really needed.

Reply:
It is also not part of the API so it should indeed be good to go